contents/handbook/strategy/strategy.md

```diff
@@ -11,27 +11,71 @@ We started by building an open-source product analytics platform with all the ba
+
+
+### How should we prioritize between competing directions?
+
+While there is value in the items on the right, we value the items on the left more
+
+**Breadth vs depth of capabilities**
+
+* **Breadth:** Build basic versions of every feature needed rather than a small set of sophisticated ones, so our customers can consolidate
+
+**High Quality Users vs High paying businesses**
+
+* **High Quality Users:** Focus on acquiring more high quality users over big ticket contracts to get better feedback and learn faster
+
+**Plugin based platform vs one-stop-shop**
+
+* **Plugin based:** To move faster, look for ways to integrate with the best tools to solve customer problems over building every feature ourselves
+
+**Self-hosted vs Cloud**
+
+* **Self-hosted:** Focus on customers who need to self-host, there’s a huge untapped market and we’re uniquely placed to win
+
+**Reject the [“modern data stack”](https://www.analytics8.com/blog/what-is-the-modern-data-stack-and-why-should-you-be-excited-about-it/) vs adapt to it**
+
+* **Reject:** We have a single product for ingesting, storing and analyzing data, we create more value by making this easy to self-host than by being a small part of a cloud stack
```

Comment:
This seems to be slightly at odds with "Plugin-based platform" – how do we decide when a feature should be part of our single product and when to instead integrate with some other best tools?

Reply:
Thanks for the feedback, this does sound contradictory - I created a new suggestion and tagged you.